### ISSUE TITLE
upgrade declarations to Laravel 5.2 

### ISSUE THREAD

**Issue by imagra** [potsky/laravel-localization-helpers#16]
By trying to upgrade to L5.2 ```artisan clear-compiled```  returns errors like this
```
Declaration of Potsky\LaravelLocalizationHelpers\Commands\LocalizationAbstract::info() 
should be compatible with Illuminate\Console\Command::info($string, $verbosity = NULL) 
```

The errors can be fixed by changing declarations to:

```
public function line( $s , $style = NULL, $verbosity = NULL) {...
public function info( $s ,  $verbosity = NULL) {...
public function comment( $s , $verbosity = NULL ) {...
public function question( $s , $verbosity = NULL) {...
```

**Comment by potsky** [comment]
Hi,

I have just released blind version 1.3.2 with a fix without any possibility to test during holidays. Could you tell me if problem is resolved now?

**Comment by imagra** [comment]
Thank you, there are no errors now with 1.3.2
